doc/user/zebra.rst

```diff
@@ -879,7 +880,8 @@ FPM Commands
 .. index:: clear fpm counters
 .. clicmd:: clear fpm counters
 
-   Resets all FPM counters.
+   Reset statistics related to the zebra code that interacts with the
```

Comment:
is this command the same as `clear zebra fpm stats`?

Reply:
Yes and no. It implements the same functionality, but for the new data plane module (which uses different command to avoid conflict).

Reply:
Because documentation says the same :) I'm just a bit confused reading it :)

Reply:
@ton31337 this is how it looks after generating the HTML:

![duplicated-doc](https://user-images.githubusercontent.com/205063/79994087-0cf3dc80-84a5-11ea-8c62-7b1aca2c528f.png)

The two commands are separated by section... I hope that is clear enough to make the differentiation.

Reply:
:+1: